frontend/src/pages/LoginPage.js

```diff
@@ -0,0 +1,85 @@
+import Head from '../components/Head';
+import Footer from '../components/Footer';
+import Navlink from '../components/Navlink';
+import FormTemplate from '../components/FormTemplate';
+import { useState, useEffect } from 'react';
+import { Form, Container, Row, Button } from 'react-bootstrap';
+
+export default function LoginPage() {
+  const [ email, setEmail ] = useState('');
+  const [ password, setPassword ] = useState('');
+  const [ btnActive, setBtnActive ] = useState(false);
+
+  // control the functionality of submit button
+  useEffect(() => {
+    if((email&& password) !== "") {
+      setBtnActive(true);
+    } else {
+      setBtnActive(false);
+    }
+  }, [ email, password ]);
+
+  function loginUser(e) {
+    e.preventDefault();
+  };
+
+  return (
+    <>
+      <Head title="Login | E-Learning System" />
+
+      <nav className="navbar navbar-expand-lg navbar-light bg-white border-bottom border-dark wrapper">
+        <a className="navbar-brand font-weight-bold" href="/">E-Learning System</a>
+        <button
+          className="navbar-toggler"
+          type="button"
+          data-toggle="collapse"
+          data-target="#navbarNavDropdown"
+          aria-controls="navbarNavDropdown"
+          aria-expanded="false"
+          aria-label="Toggle navigation">
+          <span className="navbar-toggler-icon"></span>
+        </button>
+        <div className="collapse navbar-collapse" id="navbarNavDropdown">
+          <ul className="navbar-nav">
+            <Navlink navClass="nav-link navLink active font-italic" icon={<i className="bi bi-door-open pr-1"></i>} navLink="/login" navName="Login" />
+            <Navlink navClass="nav-link navLink" icon={<i className="bi bi-person-plus pr-1"></i>} navLink="/register" navName="Register" />
+          </ul>
+        </div>
+      </nav>
```

Comment:
I noticed these are being used for `home`, `register`, and `login` page. Usually what we do when components are completely identical we just create a separate component for it so then we can just import and the changes we makes affects all pages. This also help reduce duplication of code.

for this exmaple we'd separate it in another file: `/shared/AuthHeader/AuthHeader.js` (This is just another example you can name it as you feel is proper.

Then store this shareable component there and just import the component on those 3 main pages.

Then I notice you have dynamic active links. react-router-dom have similar dynamic display function so you can use that to make your display for the active route dynamic. is called `<NavLink></NavLink>`. More detailed explanation here for you to experiment.
https://reactrouter.com/web/api/NavLink